src/db-service.ts

```diff
@@ -107,4 +126,23 @@ export class ChatService {
       )
       .then(({ rows }) => rows[0]);
   }
+
+  /**
+   * Remove all messages for the given chat
+   * @param chatId Id of chat
+   */
+  async clearChat(
+    chatId: string,
+    transactionHandler: TrxHandler,
+  ): Promise<ChatMessage[]> {
+    return transactionHandler
+      .query<ChatMessage>(
+        sql`
+            DELETE
+            FROM chat_message
+            WHERE chat_id = ${chatId} RETURNING ${ChatService.allColumns}
```

Comment:
@pyphilia Do you think i should keep the `RETURNING` or should i remove it since we discussed we would not send back deleted messages (also not recording them in actions) ?

Reply:
You can still return the rows in the api, it might be a good way to know in the task if you actually deleted something or not. But you don't have to send this result to the client.